### PR TITLE
Feat/add local qna pair

### PR DESCRIPTION
**Pull request by cachho**
Hey, first of all cool project.

I'm not sure if you want this to be an online only project. I totally see where that comes in handy. I prefer to do the loading of the data myself, then sanitize it locally, make sure everything is as I want it, and then have it embedded. I guess the difference in approach is that I want to supply my own learning material, while the online approach is more about teaching the right public material.

My specific use case is the following: I'm looking to switch from a classic QnA bot to a LLM. That's why I added the option to add a local QnA pair.

I made a new method `add_local`, to make it clear that it's not a URL you're supposed to pass to the function. This might seem like it adds complexity now, but I think this split makes sense going forward. We're going to want to add more local options, for instance regular plain text. And then we could also consider adding local variants of the online stuff, maybe not videos but local pdf makes sense.

I hope you approve of my idea, and agree that this functionality is the right way to move forward. Thanks.

### CODE REVIEW

**Comment by cachho** [comment]
I want to add that the prompt engineering that I do using `Q:... \nA:...` is how openai does it if you select the Q&A preset in the playground.

I'm happy to change the chunk size. I just looked at what you used for a pdf and decided it should be smaller since QnA pairs are usually smaller than a PDF I'd say, but that's totally negotiable.

File: embedchain/loaders_local/qna_pair.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class QnaPairLoader:
```
Comment (taranjeet) [inline embedchain/loaders_local/qna_pair.py:1]: can we please rename the loader as per comment in the issue?

**Review by taranjeet — COMMENTED** [review]
Hey, your PR looks good.
There are few changes around naming. 
Let me know your thoughts on the same.

**Comment by cachho** [comment]
Fixed the mentioned issues. I also tested it again. Works.

**Comment by taranjeet** [comment]
Thank you @cachho . This is a great contribution.

**Pull request merged**